curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/657e18b58d9f6a7ac1544999.md

```diff
@@ -0,0 +1,457 @@
+---
+id: 657e18b58d9f6a7ac1544999
+title: Step 75
+challengeType: 0
+dashedName: step-75
+---
+
+# --description--
+
+If the user has rolled both a pair and `Three of a kind`, then they have received a `Full house` resulting in `25` points.
+
+Add an `if` statement to check if both `hasThreeOfAKind` and `hasPair` are `true`. If so, call the `updateRadioOption` and pass in the numbers `2` and `25`. 
+
+# --hints--
+
+You should have an `if` statement.
+
+```js
+assert.match(code, /if\s*\(/)
+```
```

Comment:
This test will always pass. It might be better to remove this test in favor of the ones below.